### ISSUE TITLE
Newline   @"\\" and  @\longrightarrow  breaks

### ISSUE THREAD

**Issue by crvenizg** [ForNeVeR/wpf-math#267]
`Stringbuilder postupak; double mean, sumaFi;
//few lines of code
postupak.Append(mean);
postupak.Append(@"\\ "); //it breaks here, without it works correctly
postupak.Append(@"Q_{1} = \frac{\sum_{i}^N f_{i}}{4} = \frac{");
postupak.Append(sumaFi);
postupak.Append(@"}{4} = ");
postupak.Append(sumaFi / 4);
postupak.Append(@" \longrightarrow Q_{1} = "); //it breaks here, without it works correctly`

Am i using commands wrong or is it a bug ?

**Comment by ForNeVeR** [comment]
Hi! Your usage of `\\` looks fine, but we have no `longrightarrow` command, unfortunately.

A version with working `\\` wasn't published yet, unfortunately. I think I'll publish a release with it, soon.

**Comment by crvenizg** [comment]
oh you didnt publish a version with working newline \\ yet .,.. weird, read the posts before and seemed it was implemented ... ok i ll wait then ... but where is the documentation of working latex commands in this library ?

**Comment by ForNeVeR** [comment]
I have already published 0.9.0 with the fixed `\\`, a few hours ago.

For supported TeX symbols, you could skim through https://github.com/ForNeVeR/wpf-math/blob/25daf84814f47d6060b568637341153d5d273036/src/WpfMath/Data/PredefinedTexFormulas.xml and https://github.com/ForNeVeR/wpf-math/blob/25daf84814f47d6060b568637341153d5d273036/src/WpfMath/Data/TexSymbols.xml

Though I suddenly (!) realized that it's a very good idea to publish a list of supported symbols somewhere. Will create a separate issue, thanks!

**Comment by crvenizg** [comment]
Yes that would be helpful :) ... awesome newline \\ working now ... one more thing, how do u add space in latex ?

**Comment by ForNeVeR** [comment]
Try `\,`
